### ISSUE TITLE
PulsarSQL worker node readiness and liveness probes fail

### ISSUE THREAD

**Issue by robparrott** [datastax/pulsar-helm-chart#87]
The readiness and liveness probes fail for the pulsarSQL worker nodes.

For example, from the pod in question:
```
pulsar@pulsar-sql-worker-84b789f6bc-wtztr:/pulsar/conf$ hostname -i
192.168.14.34
```

Calling the status endpoint as localhost fails:
```
pulsar@pulsar-sql-worker-84b789f6bc-wtztr:/pulsar/conf$ curl http://localhost:8090/v1/service/presto
<html>
<head>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
<title>Error 404 Not Found</title>
</head>
<body><h2>HTTP ERROR 404 Not Found</h2>
<table>
<tr><th>URI:</th><td>/v1/service/presto</td></tr>
<tr><th>STATUS:</th><td>404</td></tr>
<tr><th>MESSAGE:</th><td>Not Found</td></tr>
<tr><th>SERVLET:</th><td>org.glassfish.jersey.servlet.ServletContainer-5f6e2ad9</td></tr>
</table>

</body>
</html>
```

As does calling the status endpoint via the pod IP:
```
pulsar@pulsar-sql-worker-84b789f6bc-wtztr:/pulsar/conf$ curl http://192.168.14.34:8090/v1/service/presto
<html>
<head>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
<title>Error 404 Not Found</title>
</head>
<body><h2>HTTP ERROR 404 Not Found</h2>
<table>
<tr><th>URI:</th><td>/v1/service/presto</td></tr>
<tr><th>STATUS:</th><td>404</td></tr>
<tr><th>MESSAGE:</th><td>Not Found</td></tr>
<tr><th>SERVLET:</th><td>org.glassfish.jersey.servlet.ServletContainer-5f6e2ad9</td></tr>
</table>

</body>
</html>
```

**Comment by michaeljmarshall** [comment]
@robparrott - thanks for the detailed issue report. I am looking into it now.

**Comment by michaeljmarshall** [comment]
Update: I've reproduced the issue in my local environment using minikube and the latest released version of the helm chart 2.0.1 and a slightly modified version of `examples/dev-values-sql.yaml` for my values. I am investigating a fix. I will follow up on this tomorrow.

**Comment by michaeljmarshall** [comment]
@robparrott - chart version 2.0.7 includes the fix for this issue. Please let us know if you encounter any other issues on the chart. Thanks!
